docs/explanation/archive.rst

```diff
@@ -0,0 +1,348 @@
+  in the :ref:`ArchivePockets_Release` *pocket* before they get uploaded to the
+  :ref:`ArchivePockets_Updates` *pocket*.
+| Before the initial release of an :term:`Ubuntu` :ref:`ArchiveSeries`, this
+  *pocket* contains non-security-related updates to :term:`Packages <Package>`
+  in the :ref:`ArchivePockets_Release` *pocket* before they get uploaded to the
+  :ref:`ArchivePockets_Release` *pocket*.
+
+.. _ArchivePockets_Backports:
+
+backports
+~~~~~~~~~
+
+This *pocket* contains :term:`Packages <Package>` the :term:`Ubuntu` :ref:`ArchiveSeries`
+was initially **NOT** released with.
+
+The article :doc:`/explanation/backports` provides more information on backporting software.
+
+.. important::
+
+    The *backports pocket* does not come with any security support guarantee.
+    The :term:`Ubuntu` Security Team does not update :term:`Packages <Package>`
+    in the *backports pocket*. The :term:`Ubuntu` community is responsible for
+    maintaining packages in backports with later patches for bug fixes and
+    security updates.
+
+.. _ArchiveSuite:
+
+Suite
+-----
+
+A combination of a :ref:`ArchiveSeries` and a :ref:`ArchivePockets`. For example:
+
++---------------------+----------------------+---------------------------------+
+| *Suite*             | :ref:`ArchiveSeries` | :ref:`Pocket <ArchivePockets>`  |
++---------------------+----------------------+---------------------------------+
+| ``jammy``           | ``jammy``            | :ref:`ArchivePockets_Release`   |
++---------------------+----------------------+---------------------------------+
+| ``jammy-security``  | ``jammy``            | :ref:`ArchivePockets_Security`  |
++---------------------+----------------------+---------------------------------+
+| ``jammy-updates``   | ``jammy``            | :ref:`ArchivePockets_Updates`   |
++---------------------+----------------------+---------------------------------+
+| ``jammy-proposed``  | ``jammy``            | :ref:`ArchivePockets_Proposed`  |
++---------------------+----------------------+---------------------------------+
+| ``jammy-backports`` | ``jammy``            | :ref:`ArchivePockets_Backports` |
++---------------------+----------------------+---------------------------------+
+
+You can see all active suites here: http://archive.ubuntu.com/ubuntu/dists/
+
+.. note::
+
+    The ``devel`` :ref:`ArchiveSeries` allways mirrors the :ref:`ArchiveSeries` with
+    the :term:`Codename` of the :term:`Current Release in Development`.
+
+.. _ArchiveComponents:
+
+Components
+----------
+
+*Components* are logical subdivisions or namespaces of the :term:`Packages <Package>`
+in a :ref:`ArchiveSuite`. The :term:`APT` :term:`Package Manager` can individually
+subscribe to the *components* of a :ref:`ArchiveSuite`.
+
+The :term:`Packages <Package>` of an :term:`Ubuntu` :ref:`ArchiveSeries` are categorized
+if they are :term:`Open Source Software` and part of the Base :term:`Packages <Package>`
```

Comment:
The components section was one of the harder sections to write. "base packages" was the term used here: https://ubuntu.com/about/release-cycle

There is not really a good definition. It's basically just the packages that Canonical decided to support, because they are fundamental for Ubuntu.

Reply:
I think adding that should actually be enough to make it clear, i.e. "base packages, which are fundamental for Ubuntu". The reason I don't like "base" is because it has multiple potential meanings and so has the potential to be ambiguous if we haven't defined it, but adding the "fundamental" part sort of removes that ambiguity (imo).

Reply:
Resolved with eba8fc32ad870061b15e2174da97f7c800ea1ea9, I added a term definition